docs/UserGuide.md

```diff
@@ -1,42 +1,281 @@
+5. On opening, Apollo should greet you with this:
+
+```
+____________________________________________________________
+Hello from
+ ____    ____    _____  __      __       _____
+|  _  | |  _ \  | ___ | | |     | |     | ___ |
+| |_| | | |_| | | | | | | |     | |     | | | |
+| | | | |  __/  | |_| | | |___  | |___  | |_| |
+|_| |_| |_|     \_____/ |_____| |_____| \_____/
+Enter "help" to see a list of commands.
+____________________________________________________________
 
-1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
+```
 
-## Features 
+## Command Summary
 
-{Give detailed description of each feature}
+|    Action     |              Format              |
+|:-------------:|:--------------------------------:|
+|     Help      |              `help`              |
+|  List Tasks   |              `list`              |
+|     Todo      |           `todo TASK`            |
+|   Deadline    |     `deadline TASK /by DATE`     |
+|     Event     | `event TASK /from DATE /to DATE` |
+|     Mark      |            `mark IDX`            |
+|    Unmark     |           `unmark IDX`           |
+|  Delete Task  |           `delete IDX`           |
+|     Find      |          `find KEYWORD`          |
+|     Date      |           `date DATE`            |
+|      Bye      |              `bye`               |
+|  Add Module   |       `addmod MODULE_CODE`       |
+| Delete Module |           `delmod IDX`           |
+| List Modules  |            `listmod`             |
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+> Notes about the command format:
+> + Words in `UPPER_CASE` are the parameters to be supplied by the user.
+> > e.g. in 'todo TASK', `TASK` is a parameter that can be used as `todo read book`.
+> + The `date` command only considers tasks when `DATE` is input in the format `yyyy-MM-ddThh:mm`.
+> > e.g. `deadline read book /by 2023-10-30T23:59` sets a deadline for Oct 20 2023, 11:59PM
+> + By default, all newly added tasks are not completed.
 
-Format: `todo n/TODO_NAME d/DEADLINE`
+## Features
 
-* The `DEADLINE` can be in a natural language format.
-* The `TODO_NAME` cannot contain punctuation.  
+### `help` - Viewing help
 
-Example of usage: 
+Shows a menu of commands available in Apollo and their usage, as well as their required format/parameters.
 
-`todo n/Write the rest of the User Guide d/next week`
+Format: `help`
 
-`todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
+### `list` - Listing all saved tasks
 
-## FAQ
+Shows a numbered list of all tasks (Todos, Events, Deadlines) in Apollo.
+
+Format: `list`
+
+Sample output:
+
+```
+>> list
+Here are the tasks in your list:
+1.[T][ ] Eat Lunch
+2.[E][ ] holiday (from: Mar 25 2023, 12:00AM to: Mar 30 2023, 11:59PM)
+3.[D][ ] submit tutorial (by: Mar 30 2023, 11:59PM)
+4.[T][ ] Feed the fish
+```
```

Comment:
I have updated the list command to print taskcount. Perhaps you could add that in too.

Reply:
Done

Reply:
Noted. I will proceed with the merging of pull request.